vtex-apps/my-account
# Fix address 'receiverName' field

#### What did you change? \*

Removed `receiverName` extraction from `...addressFields` for address creation handler.

#### Why? \*

The previous implementation was not sending the `receiverName` value to the mutation.

#### How to test it? \*

https://felipe--recorrenciaqa.myvtex.com/account#/addresses
Create a new user and then add a new address to it. Also, attempt to use a different value than the auto-filled one (for double-check).

#### Related to / Depends on?

https://vtex-dev.atlassian.net/browse/POST-415

#### Types of changes \*
